docs/admin/dbedit.md

```diff
@@ -0,0 +1,287 @@
+        "device": "03532e41f85f7e211eac104e3c4096d7",
+        "node": "fb3862d51210e7fc5aff87a7aba87b63",
+        "volume": "6955761dd774ceec809a9dce927da6e8",
+        "size": 5242880
+      },
+      "TpSize": 5242880,
+      "PoolMetadataSize": 28672,
+      "Pending": {
+        "Id": ""
+      }
+    }
+  },
+  "nodeentries": {
+    "fb3862d51210e7fc5aff87a7aba87b63": {
+      "State": "online",
+      "Info": {
+        "zone": 1,
+        "hostnames": {
+          "manage": [
+            "glusterblockcluster1node2.obnox_vagrant_dev.rastarnet"
+          ],
+          "storage": [
+            "192.168.55.14"
+          ]
+        },
+        "cluster": "36ac90baf26e1965feb15fe1d6e302c6",
+        "id": "fb3862d51210e7fc5aff87a7aba87b63"
+      },
+      "Devices": [
+        "03532e41f85f7e211eac104e3c4096d7"
+      ]
+    }
+  },
+  "deviceentries": {
+    "03532e41f85f7e211eac104e3c4096d7": {
+      "State": "online",
+      "Info": {
+        "name": "/dev/vdb",
+        "storage": {
+          "total": 524288000,
+          "free": 519016448,
+          "used": 5271552
+        },
+        "id": "03532e41f85f7e211eac104e3c4096d7"
+      },
+      "Bricks": [
+        "8435071a6acbfd905c00ff9ee627d940"
+      ],
+      "NodeId": "fb3862d51210e7fc5aff87a7aba87b63",
+      "ExtentSize": 4096
+    }
+  },
+  "blockvolumeentries": {},
+  "dbattributeentries": {
+    "DB_CLUSTER_HAS_FILE_BLOCK_FLAG": {
+      "Key": "DB_CLUSTER_HAS_FILE_BLOCK_FLAG",
+      "Value": "yes"
+    },
+    "DB_GENERATION_ID": {
+      "Key": "DB_GENERATION_ID",
+      "Value": "c9e784e354e040d60b1d98efe8c3d419"
+    },
+    "DB_HAS_PENDING_OPS_BUCKET": {
+      "Key": "DB_HAS_PENDING_OPS_BUCKET",
+      "Value": "yes"
+    }
+  },
+  "pendingoperations": {}
+}
+```
+
+## import command
+import creates a db file from JSON input
+
+```
+# heketi db import [flags]
+```
+
+### flags
+
+```
+--dbfile string     File path for db to be exported
+--debug             Show debug logs on stdout
+-h, --help          help for export
+--jsonfile string   File path for JSON file to be created
+```
+
+dbfile and jsonfile flags are mandatory and the path for db file should not exist.
+
+Examples
+
+```
+# heketi db import --jsonfile=heketi-db.json --dbfile=heketi-new.db  
+DB imported to heketi-new.db
+```
+
+This will create a db file with contents that reflect the contents of the json file.
+
+## delete-bricks-with-empty-path command
+delete-bricks-with-empty-path command removes brick entries from db that have empty path. It also adds back free space into the device from which the brick is removed. In earlier versions of heketi if heketi process was killed during creation of a volume, it would have left bricks that were created but not part of any volume in heketi db. To clean such bricks you may use this command. To identify such bricks use the following command:
```

Comment:
Add "The" to the beginning of the sentence.

Reply:
* from db -> from the db
* if heketi process -> if the heketi process
* in heketi db -> in the heketi db